californium-core/src/main/java/org/eclipse/californium/core/coap/LinkFormat.java

```diff
@@ -39,13 +39,12 @@
 	public static final String LINK                  = "href";
 
 	// for Resource Directory
-	public static final String HOST		     		 = "h";
```

Comment:
I don't know if this is an issue but the comment is called `Fix tab and space mixup` and the constant `HOST` was removed.

Reply:
Yeah I missed that. `HOST` is supposed to be removed in the other commit. I pushed a fix already. If its okay, I'll squash the commits to 2 commits again.